src/components/InputBar/InputBar.jsx

```diff
@@ -23,9 +20,7 @@ const mapDispatchToProps = dispatch => ({
 
 });
 
-const InputBar = connect(
+export default connect(
```

Comment:
@broneks 
I'm going to nitpick your statement -- are you stating an opinion :
> it **isn't** less readable`

or are you telling the world it shouldn't be less readable for them 

🙀